DeviceSDKAndroid/src/main/java/org/astarteplatform/devicesdk/android/AstarteAndroidMutualSSLSocketFactory.java

```diff
@@ -20,6 +21,8 @@
 
 class AstarteAndroidMutualSSLSocketFactory extends SSLSocketFactory {
   private SSLSocketFactory internalSSLSocketFactory;
+  private static Logger logger =
+      Logger.getLogger(AstarteAndroidMutualSSLSocketFactory.class.getName());
```

Comment:
Remove logger field, use Android Log API.
``` Java
private static final String TAG = "AstarteAndroidMutualSSLSocketFactory";
```

Reply:
Changed to "AndroidMutSSLSocketFact" because of the 23-character limit.